src/openforms/contrib/microsoft/client.py

```diff
@@ -47,11 +47,16 @@ class MSGraphUploadHelper:
 
     # TODO wrap upload_()-variations in single function and auto-detect object type
 
-    def __init__(self, client: MSGraphClient):
+    def __init__(self, client: MSGraphClient, options: dict):
```

Comment:
```suggestion
    def __init__(self, client: MSGraphClient, options: dict):
```
a `TypedDict` or dataclass for the options would be nice(r) for the type hinting